### PR TITLE
Apple RGB to sRGB color conversion

### LINKED ISSUE #399: Add color correction
Ran into this with another Mac port project:
https://macintoshgarden.org/forum/glider-pro-ported-windows#comment-71268

Old Mac monitors have slightly different phosphor colors and especially a different gamma curve from typical sRGB monitors.  The result is that using identical RGB values on an sRGB display doesn't produce the intended look.  In particular, reds are less saturated and midtones tend to be too dark and oversaturated.

Need to add Apple RGB -> sRGB color and gamma remapping to the fragment shader.

### PR DESCRIPTION
**Pull request by elasota**
Fixes #399 

Mostly increases brightness.

Before: 
image

After:
image

### CODE REVIEW

File: src/video/glsl/fragment.frag [hunk 1]
```diff
@@ -85,4 +106,5 @@ void main() {
             frag_color = vec4(0, 0, 0, 0);
         }
     }
+	frag_color.rgb = apple_rgb_to_srgb(frag_color.rgb);
```
Comment (sfiera) [inline src/video/glsl/fragment.frag:109]:
```suggestion
    frag_color.rgb = apple_rgb_to_srgb(frag_color.rgb);
```

**Review by sfiera — COMMENTED** [review]
This is changing some of the outputs that are tested, so you’ll need to need to update the expected images:
```sh
out/cur/offscreen main-screen -o test/main-screen
out/cur/offscreen options -o test/options
out/cur/build-pix -o test/build-pix
```

**Comment by sfiera** [comment]
It’s also possible that this change will make it impossible to test png outputs altogether. The replays stopped testing images a long time ago because scaled sprites didn’t scale exactly the same on different GPUs (those tests rely on a log of drawing operations now). I kept a couple of png-based tests with the goal of really testing the renderer, but it wouldn’t surprise me if the RGB mapping varies ever so slightly across GPUs as well.

**Comment by sfiera** [comment]
> This is changing some of the outputs that are tested, so you’ll need to need to update the expected images

Ah, but you didn’t implement offscreen video on Windows yet, right? I’ll pull this and try.

**Comment by elasota** [comment]
Offscreen isn't implemented in Windows yet, no.  If this fails the PNG tests then might need to add some +/-1 tolerances or something.  I don't think this should ever be off by more than that unless the shader is compiling low precision.

**Comment by sfiera** [comment]
Looks like it’s as I feared. The simpler solution will be to switch those tests to run against text data for now. That will unblock this PR and I can look at re-enabling image diffs separately. PR momentarily.

**Comment by sfiera** [comment]
OK, if you rebase this PR, then pull `/test` again, that should make tests pass.

**Pull request merged**